packages/hooks/src/check-update.js

```diff
@@ -0,0 +1,330 @@
+#!/usr/bin/env node
+import fs from 'fs';
+import util from 'util';
+import childProcess from 'child_process';
+
+const exec = util.promisify(childProcess.exec);
+
+const SLACK_BOLT_SDK = '@slack/bolt';
+const SLACK_CLI_HOOKS = '@slack/hooks';
+const SLACK_DENO_SDK = '@slack/deno-slack-sdk';
```

Comment:
Can't we remove this, as well as the subsequent references to it?

Reply:
My bad. I'm sure I mistakenly left it in. Let's pull it (and verify all works as expected, ofc).